src/js/ws-client.js

```diff
@@ -0,0 +1,43 @@
+var RiseVision = RiseVision || {};
+RiseVision.Common = RiseVision.Common || {};
+
+RiseVision.Common.WSClient = (function() {
+
+  function broadcastMessage(message) {
+    try {
```

Comment:
Maybe create a `safeWrite` function to avoid repeating this code:

```javascript
function saveWrite(message) {
   try {
      if (top.RiseVision.LocalMessaging) {
         top.RiseVision.LocalMessaging.write(message);
      }
   } catch (err) {
       console.log( "widget-common: ws-client", err );
   }
}

function broadcastMessage(message) {
   safeWrite(message);
}

function getModuleClientList() {
   safeWrite({ topic: "client-list-request" });
}

```